libflux/src/semantic/tests.rs

```diff
@@ -208,39 +185,47 @@ macro_rules! test_infer_err {
                     .fold(String::new(), |acc, (name, poly)| acc
                         + &format!("\t{}: {}\n", name, poly))
             )
-        };
+        }
     }};
     ( src: $src:expr $(,)? ) => {{
-        let env: Vec<(&str, &str)> = Vec::new();
+        let env = HashMap::new();
```

Comment:
Can just inline this.

Reply:
I could, but I'm already having enough difficulty with reading the long lines. Rust seems to be a more horizontal language than I am used to.